bazel/repository_locations.bzl

```diff
@@ -89,12 +89,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Abseil",
         project_desc = "Open source collection of C++ libraries drawn from the most fundamental pieces of Google’s internal codebase",
         project_url = "https://abseil.io/",
-        version = "5d8fc9192245f0ea67094af57399d7931d6bd53f",
-        sha256 = "e3812f256dd7347a33bf9d93a950cf356c61c0596842ff07d8154cd415145d83",
+        version = "a9831f1cbf93fb18dd951453635f488037454ce9",
+        sha256 = "8a144e21e0110903a1664dd89cabc327ce26bc56e04e02d49335544b9a64631f",
         strip_prefix = "abseil-cpp-{version}",
         urls = ["https://github.com/abseil/abseil-cpp/archive/{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        release_date = "2020-11-24",
+        release_date = "2021-05-03",
```

Comment:
Now that Abseil has regular releases, can / should we switch to the releases? https://github.com/abseil/abseil-cpp/releases

Reply:
I'm not sure about this. Google just has a single master version in our monorepo, so not sure how the versions are decided.

Reply:
The repo recommends 'live-at-head' but document their release strategy at https://github.com/abseil/abseil-cpp#releases and https://abseil.io/about/releases

> NOTE: we don’t want you to think of these snapshots as “versions.” They are simply a snapshot of the codebase at a specific point in time.

Shall we stick with commit versions and 'live-at-head' for abseil @htuch ?

Reply:
I think we prefer releases as an abstract handle, but if there are good functional or other reasons to not in a project with this philosophy, we can override.